src/main/host/descriptor/epoll/key.rs

```diff
@@ -56,7 +56,7 @@ impl Eq for PriorityKey {}
 
 impl PartialEq for PriorityKey {
     fn eq(&self, other: &Self) -> bool {
-        self.key.eq(&other.key)
+        self.pri.eq(&other.pri)
```

Comment:
Add a comment abut why you only need to compare by the priority and not the key? I think it's because no two `PriorityKey`s should ever have the same priorities? Maybe add this restriction to the docs for `PriorityKey::new` as well?